.github/actions/setup/action.yml

```diff
@@ -7,7 +7,7 @@ runs:
       uses: pnpm/action-setup@v2
       with:
         version: 8.6.5
-    - name: Setup node 16.x
+    - name: Setup node
```

Comment:
Remove it completely rather than fixing it so we don't have to think about it staying in sync